### DISCUSSION TITLE
🐰coderabbit について

### DISCUSSION THREAD

**Discussion by habvi** [42-pong/42-pong#55] [Tips]
- 無料版は PR の summary 作成のみ
- 最初の 14 日間だけ有料版の機能 (PR review など) も使える
- PR review 機能
  - Draft PR だと coderabbit は呼ばれない
  - ready for review 状態にすると自動で coderabbit が呼ばれる場合 / 呼ばれない場合がある (merge 先の設定など…？)
    - 自動で呼ばれない場合、手動で PR comment に `@coderabbitai review` と打って送るとしてくれる
  - review に返信すると追加質問などができる
  - 追加情報がなくても「はい、分かりました。」みたいな返信だけ来る

**Comment by ayase-mstk** [comment]
> ready for review 状態にすると自動で coderabbit が呼ばれる場合 / 呼ばれない場合がある (merge 先の設定など…？)

現在のcoderabbitの修正PRがマージされれば必ず呼ばれるようになるかと思います！

> Draft PR だと coderabbit は呼ばれない

Draft PRでも呼ばれるようにした方がいいですか？

**Reply by habvi** [reply]
ありがとうございます！

Draft PR は、個人的にはレビューされる準備が整ってない場合も多いので、今のまま呼ばれないで大丈夫な気がします！
